pkg/grpc/device_definition.proto

```diff
@@ -151,6 +151,8 @@ message Integration {
     int32 ICE = 3;
     int32 PHEV = 4;
   }
+  optional int64 points = 9;
+  optional uint64 manufacturer_token_id = 10;
```

Comment:
It looks like in this neck of the woods they don't do optional. Maybe that's even correct. Let's take it off.